clients/web/src/lib/components/organisms/scrims/modals/PlayerManagementModal.svelte

```diff
@@ -0,0 +1,104 @@
+<script lang="ts">
+    import {Modal} from "$lib/components";
+    import {createRestrictionMutation, getMemberByUserIdQuery} from "$lib/api";
+    import FaHammer from "svelte-icons/fa/FaHammer.svelte";
+
+    export let visible = false;
+
+    let buttonEnabled = true;
+    export let playerId:number;
+
+
+    async function createBan() {
+        buttonEnabled = false;
+        // toast
+        // remove ban
+        try {
+
+            const memberRequest = (await getMemberByUserIdQuery({id: playerId, orgId: 1}));
+            console.log(memberRequest);
+            const memberId = memberRequest.id ?? 0;
+            createRestrictionMutation({memberId: memberId, reason:"Because nigel said yo", expiration: new Date(Date.now() + 1000*60*60*12)})
```

Comment:
Need to add a field for the reason, so these aren't just because nigel said; also if we can add a comment stating what the expiration is.